tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SDK/ILambdaContext.cs

```diff
@@ -0,0 +1,91 @@
+// <copyright file="ILambdaContext.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+using System;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AWS.SDK;
+
+/// <summary>
+/// Object that allows you to access useful information available within
+/// the Lambda execution environment.
+/// </summary>
+internal interface ILambdaContext
```

Comment:
Are we using all the fields from this file? if not, shouldn't we remove the ones we are not referencing?

Reply:
Yeah... i think it's a good idea. will do it.

Reply:
commented out unused properties.